src/plugins/LinkManager/createReactTooltipWrapper.tsx

```diff
@@ -0,0 +1,66 @@
+import Nanobus from "nanobus"
```

Comment:
Can't say I've used nanobus before but I think I grasp the general straws of it. Did have a question more out of curiosity that feedback (so no rush on responding) I'm a little hazy on the scope of the listener (in this case "update") - ie: is it tied to the specific component or could you emit an event anywhere in the DOM if the Wrapper was mounted or is it tied to an emitter that has to be passed in